### PR TITLE
Ca 4074 architect scripting

### PR DESCRIPTION
**Pull request by ericSpence**
Adding support to allow customers to use Architect-Scripting in the code-editor. Architect scripting allows users to create call/automate flows through scripts rather than through the UI. 
Note: I am noticing some failing unit test cases but they appear to be the same ones that are failing for me in Master so they are not related to my changes. I can look at those as well if no one is already doing so.

### CODE REVIEW

**Comment by ericSpence** [comment]
Currently doc for Architect Scripting is in ```public/assets/documentation/* ```  since there is no other place that it is currently hosted. If there is a better spot for that to be hosted I would be more than happy to Iframe it in and remove all that code from this repo.

File: package.json [hunk 1]
```diff
@@ -50,5 +50,11 @@
   "engines": {
     "node": "^4.5 || 6.* || >= 7.*"
   },
-  "private": true
+  "private": true,
+  "dependencies": {
+    "architect-scripting": "git+ssh://git@bitbucket.org:inindca/architect-scripting.git",
```
Comment (KevinGlinski) [inline package.json]: You can't have a private repo referenced in a public repo.  If someone w/o inindca bitbucket access wanted to run dev tools locally they wouldn't be able to.

**Comment by KevinGlinski** [comment]
please remove public/assets/documentation/* from this pr

File: package.json (hunk 1 above)
Comment (ericSpence) [reply package.json]: That makes sense. I will look into getting a public package of it setup.

**Comment by KevinGlinski** [comment]
looks good but doc needs to be in dev center before merging

**Comment by ericSpence** [comment]
Awesome. Do you have any guidance to where in the Developer Center that you would like that doc?

**Comment by KevinGlinski** [comment]
open an api- jira ticket with the location of the jsdoc md file and i'll figure one out

**Comment by ericSpence** [comment]
Doc is up and hosted on github pages. https://mypurecloud.github.io/purecloud-flow-scripting-api-sdk-javascript/ updated codeeditor to reference that in its iframe

**Comment by ericSpence** [comment]
Is there any other work needed on this branch?

**Pull request merged**